Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -247,6 +247,25 @@ class RUMViewScopeTests: XCTestCase {
 
     // MARK: - User Action Tracking
 
+    func test_finalizeActiveUserActionsOnStopView() throws {
+        let scope = RUMViewScope(
+            parent: parent,
+            dependencies: dependencies,
+            identity: mockView,
+            attributes: [:],
+            startTime: Date()
+        )
+        XCTAssertTrue(scope.process(command: RUMStartViewCommand.mockWith(identity: mockView)))
+        let mockUserActionCmd = RUMAddUserActionCommand.mockAny()
+        XCTAssertTrue(scope.process(command: mockUserActionCmd))
+        XCTAssertFalse(scope.process(command: RUMStopViewCommand.mockWith(identity: mockView)))
+
+        let recordedActionEvents = try output.recordedEvents(ofType: RUMEvent<RUMAction>.self)
```

Comment:
Have you considered putting this test in `RUMUserActionScopeTests`? If you look closer at the assertions we do in each scope tests, the testing strategy will emerge: _"Scope X tests only assert `RUMEvents` send directly from this scope"_. This helps with tests granulation. Without a common strategy, we could be testing everything in the `RUMApplicationScopeTests`, which will lower the readability.